Roni.Corona/Roni.Corona.DataIngestion/Integrations/CsseIntegration.cs

```diff
@@ -2,42 +2,36 @@
 using System.Collections.Generic;
 using System.Net.Http;
 using System.Threading.Tasks;
+using Flurl;
```

Comment:
Let's actually not go down this path. Adding dependencies is a drama, and there is no reason at this point to use an external library for a simple integration nor will we have more integrations in this project

Reply:
This has been completed.